jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -142,4 +138,30 @@ BuildDockerTask setJibExtension(JibExtension jibExtension) {
     this.jibExtension = jibExtension;
     return this;
   }
+
+  /**
+   * Returns an {@link ImageReference} retrieved from the build configuration, or the project name/
+   * version if no target image is configured.
+   *
+   * @param gradleBuildLogger the logger used to notify users of the target image parameter
+   * @return an {@link ImageReference} parsed from the configured target image, or one of the form
+   *     "project-name:project-version" if target image is not configured
+   */
+  ImageReference getDockerTag(GradleBuildLogger gradleBuildLogger)
+      throws InvalidImageReferenceException {
+    Preconditions.checkNotNull(jibExtension);
+    if (Strings.isNullOrEmpty(jibExtension.getTargetImage())) {
+      // TODO: Validate that project name and version are valid repository/tag
+      gradleBuildLogger.lifecycle(
+          "Tagging image with generated image reference "
```

Comment:
Could add this to `HelpfulSuggestions` (Maven/Gradle specifics in `HelpfulSuggestionsProvider`), or have a TODO for now